stingray/bispectra.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python
+
+from __future__ import division
+import numpy as np
+from scipy.linalg import hankel
+import scipy.fftpack
+import stingray.lightcurve as lightcurve
+import stingray.utils as utils
+from stingray.utils import simon
+__all__ = ["bispectra"]
+
+
+class bispectra(object):
```

Comment:
I would also call the class `Bispectrum` to be consistent with nomenclature generally used in astronomy.